We force "daemonize no" in init script, so daemonize and pidfile does not
have any effect in keydb.conf. It's init/rc system's job to daemonize
programs and handle pidfiles, if it needs it!

Applications often communicate with Redis via unix socket, so it's
convenient to have it enabled by default. It must be in a subdirectory that
is automatically created by the init script because keydb user has no
permission to create files in /run.

--- a/keydb.conf
+++ b/keydb.conf
@@ -123,8 +123,8 @@
 # incoming connections. There is no default, so KeyDB will not listen
 # on a unix socket when not specified.
 #
-# unixsocket /tmp/keydb.sock
-# unixsocketperm 700
+unixsocket /run/keydb/keydb.sock
+unixsocketperm 770
 
 # Close the connection after a client is idle for N seconds (0 to disable)
 timeout 0
@@ -279,10 +279,6 @@
 
 ################################# GENERAL #####################################
 
-# By default KeyDB does not run as a daemon. Use 'yes' if you need it.
-# Note that KeyDB will write a pid file in /var/run/keydb.pid when daemonized.
-daemonize no
-
 # If you run KeyDB from upstart or systemd, KeyDB can interact with your
 # supervision tree. Options:
 #   supervised no      - no supervision interaction
@@ -295,17 +291,6 @@
 #       They do not enable continuous pings back to your supervisor.
 supervised no
 
-# If a pid file is specified, KeyDB writes it where specified at startup
-# and removes it at exit.
-#
-# When the server runs non daemonized, no pid file is created if none is
-# specified in the configuration. When the server is daemonized, the pid file
-# is used even if not specified, defaulting to "/var/run/keydb.pid".
-#
-# Creating a pid file is best effort: if KeyDB is not able to create it
-# nothing bad happens, the server will start and run normally.
-pidfile /var/run/keydb_6379.pid
-
 # Specify the server verbosity level.
 # This can be one of:
 # debug (a lot of information, useful for development/testing)
@@ -317,14 +302,14 @@
 # Specify the log file name. Also the empty string can be used to force
 # KeyDB to log on the standard output. Note that if you use standard
 # output for logging but daemonize, logs will be sent to /dev/null
-logfile ""
+# logfile "/var/log/keydb.log"
 
 # To enable logging to the system logger, just set 'syslog-enabled' to yes,
 # and optionally update the other syslog parameters to suit your needs.
-# syslog-enabled no
+syslog-enabled yes
 
 # Specify the syslog identity.
-# syslog-ident keydb
+syslog-ident keydb
 
 # Specify the syslog facility. Must be USER or between LOCAL0-LOCAL7.
 # syslog-facility local0
@@ -350,16 +335,13 @@
 #
 # However it is possible to force the pre-4.0 behavior and always show a
 # ASCII art logo in startup logs by setting the following option to yes.
-always-show-logo yes
+# always-show-logo yes
 
 # By default, KeyDB modifies the process title (as seen in 'top' and 'ps') to
 # provide some runtime information. It is possible to disable this and leave
 # the process name as executed by setting the following to no.
 set-proc-title yes
 
-# Retrieving "message of today" using CURL requests.
-#enable-motd yes
-
 # When changing the process title, KeyDB uses the following template to construct
 # the modified title.
 #
@@ -470,7 +452,7 @@
 # The Append Only File will also be created inside this directory.
 #
 # Note that you must specify a directory here, not a file name.
-dir ./
+dir /var/lib/keydb
 
 ################################# REPLICATION #################################
 
@@ -2105,4 +2087,7 @@
 # this is only exposed via the info command for clients to use, but in the future we
 # we may also use this when making decisions for replication.
 #
-# availability-zone "us-east-1a"
\ No newline at end of file
+# availability-zone "us-east-1a"
+
+# Include configuration fragments.
+include /etc/keydb/keydb.d/*.conf
